src/cmake/modules/FindOpenEXR.cmake

```diff
@@ -166,21 +166,23 @@ find_path (OPENEXR_INCLUDE_PATH OpenEXR/OpenEXRConfig.h
 # Try to figure out version number
 if (DEFINED _OPENEXR_VERSION AND NOT "${_OPENEXR_VERSION}" STREQUAL "")
     set (OPENEXR_VERSION "${_OPENEXR_VERSION}")
```

Comment:
Shouldn’t this line be removed as well?

Reply:
I'm setting both OPENEXR_VERSION and also OpenEXR_VERSION, in case somebody is depending on the old one.